usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf

```diff
@@ -1,48 +1,52 @@
 # Begin example setup for SAP HANA on UEFI capable systems.
-# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL.
-# You may activate SSH_ROOT_PASSWORD and adapt the 'password_on_the_rear_recovery_system'.
-# For basic information see the SLE11 and SLE12 manuals.
+
+# You must adapt "your.NFS.server.IP/path/to/your/rear/backup" at BACKUP_URL and also
+# the HANA related volume groups and mount points at EXCLUDE_VG and BACKUP_PROG_EXCLUDE.
+# As needed you may activate SECURE_BOOT_BOOTLOADER and USE_DHCLIENT
+# and also SSH_ROOT_PASSWORD with a 'password_on_the_rear_recovery_system'.
+# For basic information see the SUSE Linux Enterprise Server manuals.
 # Also see the support database article "SDB:Disaster Recovery"
 # at http://en.opensuse.org/SDB:Disaster_Recovery
 # In particular note:
 # There is no such thing as a disaster recovery solution that "just works".
-# SAP HANA on UEFI capable systems need EFI and DOS tools.
-# Installing DOS file system tools is optional as dosfslabel is not used within ReaR anymore thanks to issue #694
-#
-# Install EFI ISO creation tool:
-# For UEFI the ebiso package is required to use ReaR on SLES 11 and SLES 12.
-# The ebiso package is available at
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/
-# For example to download the ebiso package for SLES 12 use a command like (the RPM package version and release numbers change):
-#   wget http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/SLE_12/x86_64/ebiso-0.1.4-1.2.x86_64.rpm
-# and install it with a command like:
-#   rpm -ivh ebiso-0.1.4-7.1.x86_64.rpm
-# Furthermore ebiso needs ReaR higher than 1.17.2 e.g. use a development snapshot release from
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear:/Snapshot/
+
 # Create the ReaR recovery system as ISO image:
 OUTPUT=ISO
-# Use ebiso to make an UEFI bootable ISO image of the ReaR recovery system:
-ISO_MKISOFS_BIN=/usr/bin/ebiso
-# Store the backup file via NFS on a NFS server:
+
+# Use /usr/bin/xorrisofs (provided by the RPM package xorriso)
+# to create a UEFI bootable ISO image of the ReaR recovery system:
+ISO_MKISOFS_BIN="/usr/bin/xorrisofs"
+
+# To boot the ReaR recovery system with UEFI Secure Boot
+# specify a (properly signed) first stage bootloader for Secure Boot:
+#SECURE_BOOT_BOOTLOADER="/boot/efi/EFI/sles/shim.efi"
```

Comment:
With #3276 merged, is this still required?

Reply:
Using only USING_UEFI_BOOTLOADER=1 worked for us in our installations

Reply:
I didn't test things here, see my initial PR description.
I only liked to change it according to what I already know
what had worked for me when I had tested it.

Reply:
I mentioned it in
https://github.com/rear/rear/issues/3300#issuecomment-2317211811

Reply:
Via
https://github.com/rear/rear/pull/3306/commits/7946ee20fd56223563a3d4a107cf6470c2d00895
I added a part about setting USING_UEFI_BOOTLOADER